src/main/java/com/iotiq/application/service/CustomerService.java

```diff
@@ -55,6 +79,7 @@ public CustomerDto getCustomer() {
         customerDto.setMedicalData(Objects.requireNonNullElseGet(customer.getMedicalData(), MedicalData::new));
         customerDto.setCart(customerDto.getCart());
         customerDto.setOrders(ModelMapperUtil.map(customer.getOrders(), OrderDto.class));
+        customerDto.setOrders(setLastTwoOrders(customerDto.getOrders()));
```

Comment:
custmerDto.getOrders? It can be customer.getOrders. And why setting the orders twice?